Project.toml

```diff
@@ -9,6 +9,6 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Tokenize = "0796e94c-ce3b-5d07-9a54-7f471281c624"
 
 [compat]
-CSTParser = "^1.0.0"
+CSTParser = ">= 1.0.0"
```

Comment:
@domluna I think you want `CSTParser = "1, 2"` here. Otherwise you say that this version is compatible with all future versions of CSTParser